IdentityCore/src/oauth2/MSIDExternalSSOContext.m

```diff
@@ -24,7 +24,77 @@
 
 
 #import "MSIDExternalSSOContext.h"
+#import "MSIDWPJKeyPairWithCert.h"
+#import "MSIDKeychainUtil.h"
 
 @implementation MSIDExternalSSOContext
 
+- (MSIDWPJKeyPairWithCert *)wpjKeyPairWithCertWithContext:(id<MSIDRequestContext>)context
+{
+#if TARGET_OS_OSX
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130000
+    if (@available(macOS 13.0, *))
```

Comment:
Nit: do we need dual version checks here?

Reply:
i think it is the best practice. line 34 and 35 does the check during compile time and line 36 does the check during run time. Run time check allows us to handle code for older os version in else , although we are not doing anything for older versions.

nit : line 34 and 35 can be combined with &&, so we can replace double end if with one #endif

Reply:
Without preprocessor, older XCode will fail to compile this. Without available check, older OS will crash